src/api/ddoc/views.rst

```diff
@@ -315,6 +315,1144 @@ including the update sequence of the database from which the view was
+
+.. note::
+    The ``count`` operation works only if the indexed values are strings.
+    The indexed values cannot be mixed types. For example,
+    if 100 strings are indexed, and one number,
+    then the index cannot be used for ``count`` operations. 
+    You can check the type by using the ``typeof`` operator, and convert it by using the ``parseInt``,
+    ``parseFloat``, or ``.toString()`` functions.
+
+*Example of a query using the ``counts`` facet syntax:* 
+
+.. code-block:: http
+
+    ?q=*:*&counts=["type"]
+
+*Example response after using of the ``counts`` facet syntax:*
+
+.. code-block:: javascript
+
+    {
+        "total_rows":100000,
+        "bookmark":"g...",
+        "rows":[...],
+        "counts":{
+            "type":{
+                "sofa": 10,
+                "chair": 100,
+                "lamp": 97
+            }
+        }
+    }
+
+``drilldown``
+^^^^^^^^^^^^^
+
+.. note:: 
+    The ``drilldown`` option is only available when making global queries.
+
+You can restrict results to documents with a dimension equal to the specified label.
+Restrict the results by adding ``drilldown=["dimension","label"]`` to a search query.
+You can include multiple ``drilldown`` parameters to restrict results along multiple dimensions.
+
+Using a ``drilldown`` parameter is similar to using ``key:value`` in the ``q`` parameter,
+but the ``drilldown`` parameter returns values that the analyzer might skip.
+
+For example,
+if the analyzer did not index a stop word like ``"a"``,
+using ``drilldown`` returns it when you specify ``drilldown=["key","a"]``.
+
+Ranges
+^^^^^^
+
+.. note:: 
+    The ``ranges`` option is only available when making global queries.
+
+The ``range`` facet syntax reuses the standard Lucene syntax for ranges
+to return counts of results that fit into each specified category.
+Inclusive range queries are denoted by brackets (``[``, ``]``).
+Exclusive range queries are denoted by curly brackets (``{``, ``}``).
+
+.. note::
+    The ``range`` operation works only if the indexed values are numbers. 
+    The indexed values cannot be mixed types. For example, if 100 strings are indexed,
+    and one number, then the index cannot be used for ``range`` operations.
+    You can check the type by using the ``typeof`` operator, and convert 
+    it by using the ``parseInt``, ``parseFloat``, or ``.toString()`` functions.
+
+*Example of a request that uses faceted search for matching ``ranges``:*
+
+.. code-block:: http
+
+    ?q=*:*&ranges={"price":{"cheap":"[0 TO 100]","expensive":"{100 TO Infinity}"}}
+
+*Example results after a ``ranges`` check on a faceted search:*
+
+.. code-block:: javascript
+
+    {
+        "total_rows":100000,
+        "bookmark":"g...",
+        "rows":[...],
+        "ranges": {
+            "price": {
+                "expensive": 278682,
+                "cheap": 257023
+            }
+        }
+    }
+
+.. _api/ddoc/view/geographical_searches:
+
+Geographical searches
+---------------------
```

Comment:
Do we need a disclaimer here that this is Lucene's implementation, not the fancy geo index Cloudant also has open sourced? It probably only matters to geo-heads, but if I understand correctly, there's no map projection going on in Lucene, this is just a numerical bounding box.

Reply:
Makes sense, maybe just something subtle like 

> you can also sort your results by their distance from a geographic coordinate using Lucene's builtin geospatial capabilities.

For the typical reader it'll just seem like a bit of extra verbiage, but a geo-head will understand it's not a `libspatialindex` thing.

Reply:
Still needed.